attributes/default.rb

```diff
@@ -72,18 +72,26 @@
 default['postfix']['main']['mydomain'] = (node['domain'] || node['hostname']).to_s.chomp('.')
 default['postfix']['main']['myorigin'] = '$myhostname'
 default['postfix']['main']['mydestination'] = [node['postfix']['main']['myhostname'], node['hostname'], 'localhost.localdomain', 'localhost'].compact
-default['postfix']['main']['smtpd_use_tls'] = 'yes'
-default['postfix']['main']['smtp_use_tls'] = 'yes'
+default['postfix']['main']['smtpd_tls_security_level'] = 'may'
+default['postfix']['main']['smtp_tls_security_level'] = 'may'
 default['postfix']['main']['smtp_sasl_auth_enable'] = 'no'
 default['postfix']['main']['mailbox_size_limit'] = 0
 default['postfix']['main']['mynetworks'] = nil
 default['postfix']['main']['inet_interfaces'] = 'loopback-only'
+# With Postfix 2.3 and later use smtpd_tls_security_level instead.
```

Comment:
Please delete rather than comment here. And make a note of your changes in `CHANGELOG.md`